mysql-test/suite/sys_vars/r/all_vars.result

```diff
@@ -15,6 +15,8 @@ left join t1 on variable_name=test_name where test_name is null ORDER BY variabl
 There should be *no* variables listed below:
 DISABLED_STORAGE_ENGINES
 DISABLED_STORAGE_ENGINES
+INNODB_ENCRYPT_TABLES
```

Comment:
Please review this testcase, specifically the "There should be *no* variables listed below:" comment above :)

As you create sys_vars.innodb_encrypt_tables_basic, move the sysvar tests from the feature testcase